ffi/tests/bindgen_derive.rs

```diff
@@ -0,0 +1,60 @@
+#![allow(unsafe_code, clippy::restriction, clippy::pedantic)]
+
+use std::mem::MaybeUninit;
+
+use getset::{Getters, MutGetters, Setters};
+use iroha_ffi::ffi_bindgen;
+
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct Name(String);
+
+#[ffi_bindgen]
+#[derive(Setters, Getters, MutGetters)]
+#[getset(get = "pub")]
+pub struct FfiStruct {
+    #[getset(set = "pub", get_mut = "pub")]
+    id: u32,
+    name: Option<Name>,
+    is_null: Option<Name>,
+}
+
+#[test]
+fn getset_get() {
+    let init_name = Name("Name".to_owned());
+    let ffi_struct: *mut _ = &mut FfiStruct {
+        id: 0,
+        name: Some(init_name.clone()),
+        is_null: None,
+    };
+
+    let mut id = MaybeUninit::<*mut u32>::new(core::ptr::null_mut());
+    let mut name = MaybeUninit::<*const Name>::new(core::ptr::null());
+    let mut is_null = MaybeUninit::<*const Name>::new(core::ptr::null());
+
+    unsafe {
+        //FfiStruct__set_id(ffi_struct, 42_u32);
+        FfiStruct__id_mut(ffi_struct, id.as_mut_ptr());
+        let id = &mut *id.assume_init();
+        assert_eq!(&mut 0, id);
```

Comment:
At first I was thinking that it's a strange way to check for a null pointer. And then I realized that it's just a reference to id. Maybe use another number instead of 0?

Reply:
ok, makes sense if it confuses the reader